docs/decisions/0004-exam-ida.rst

```diff
@@ -0,0 +1,59 @@
+Exams as an Independently Deployed Application
+====================
+
+Status
+------
+
+Propsed (circa April 2022)
+
+Context
+-------
+The process to release changes in this library to a production environment is both slow and involves multiple manual steps.
+Developers must update version numbers for this library, create a tag, and merge updated requirements to edx-platform to release changes.
+Waiting for the platform CI checks and deployment pipeline to run can take up to half a day. This adds significant turnaround
+to releasing even the smallest of changes.
+
+This led us to evaluate the need for these timed/proctored exam features and data models to be available alongside edx-plaform as a plugin. This library is currently coupled with the platform without a well-defined interface leading to many imports in both directions. We must use a runtime wrapper just to avoid the circular dependencies that exist in this approach today. However, the actual data models contained within this library are specific to proctoring tools and user flow. The features within this library are generally not needed in other contexts.
+
+We believe we can create a well-defined interface that results in an exam solution that is best managed as its own service. The following proposed component diagram defines the relationship between platform components and the bounded context of timed exams.
+
+`Exam Components`_
+
+Additionally, we intend to eliminate vendor-specific integration code by implementing the IMS's `Proctoring Services Standard`_ built on top of LTI 1.3. This new IDA gives us a clean starting point to build that interface.
+
+.. _Proctoring Services Standard: https://www.imsglobal.org/proctoring-services
+
+.. _Exam Components: https://lucid.app/lucidchart/a4b40637-93f1-47f1-bb17-d68e4ff6f9d9/edit?invitationId=inv_92a6d5c4-d14c-472a-b2bc-64b64d9d45ef
+
+Decision
+--------
+A new repository and IDA will be created for the exam service. As service will handle all configuration and interfacing with third party proctoring tools. It will also be the source of truth for any state information about a learner's progress through exam setup, completion, and review flows.
```

Comment:
`As service will handle all configuration and interfacing with third party proctoring tools.`

I'm not sure what this means. Could you clarify, please?